app/assets/stylesheets/_authors.scss

```diff
@@ -1,9 +1,39 @@
 .author__name {
-  font-size: 1.5em;
+  font-size: $base-title-size;
 }
 
 .author__books {
-  @include display(flex);
-  @include border-color(null null #CCCCCC null);
+  @include border-color(null null $base-secondary-color null);
   @include border-style(null null solid null);
+  @include display(flex);
+}
+
+.author-cover {
+  @include span-columns(4);
 }
+
+.author-info {
+  @include span-columns(8);
+  @include omega();
+  span {
+    font-weight:bold;
+  }
+}
+.author-page {
+  .books{
+    clear:left;
+    @include shift(4);
+  }
+  .book {
+    @include fill-parent();
+    @include row(table);
+    .cover_image {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `cover_image` should be written in lowercase with hyphens